fvtest/tril/test/IlGenTest.cpp

```diff
@@ -50,7 +50,10 @@ TEST_F(IlGenTest, Return3) {
     auto Int32 = types.PrimitiveType(TR::Int32);
     TR::IlType* argTypes[] = { Int32 };
 
-    Tril::TRLangBuilder injector(trees, &types);
+    Tril::GenericNodeConverter genericNodeConverter;
+    Tril::CallConverter callConverter(&genericNodeConverter);
+
+    Tril::TRLangBuilder injector(trees, &types, &genericNodeConverter);
```

Comment:
I think this is supposed to be 
```suggestion
    Tril::TRLangBuilder injector(trees, &types, &callConverter);
```